phpunit/fonts-library/class-wp-font-family-test.php

```diff
@@ -0,0 +1,337 @@
+<?php
+/**
+ * Tests for Font Family Utils class
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ */
+
+/**
+ * @coversDefaultClass WP_Font_Family
+ */
+class WP_Font_Family_Test extends WP_UnitTestCase {
+
+	/**
+	 * Test the constructor failure
```

Comment:
```suggestion
	 * Tests that an exception is thrown when the font family is missing a slug.
```
- Methods should use a third-person singular verb ( `Tests` rather than `Test` ).
- Added context to the failure.